soco/core.py

```diff
@@ -238,6 +244,20 @@ def __str__(self):
     def __repr__(self):
         return '{0}("{1}")'.format(self.__class__.__name__, self.ip_address)
 
+    @classmethod
+    def any_soco(cls):
```

Comment:
Not extremely important here, but do you think maybe this would fit better under discover. I know that I have many times also wanted to have a from_name method in discover, to just always get Kitchen. So I thought maybe it would make sense to have a few helpers there under discover.